recipe/meta.yaml

```diff
@@ -5,10 +5,13 @@
 
 package:
   name: libgfortran
-  version: {{ libgfortran_version|join('.') }}
+  # This is hacked to match the higher version on defaults to make conda-forge 
+  # more compatible with defaults.
+  # The actual libs are the same.
+  version: 3.0.1
```

Comment:
Would it not be better to change the first line of the recipe: `set libgfortran_version = [3, 0, 0]`?

Reply:
I chose not to for a few reasons.

1. I wanted it to be very obvious that this recipe had been hacked.
2. I wanted to preserve the value of the variable as the true libgfortran version so that it could be used in other places. 

On defaults, they did the same thing to the recipe:

```yaml
{% set libgfortran_version = [3, 0, 0] %}
{% set libquadmath_version = [0, 0, 0] %}
{% set libgcc_s_version = [1, 0, 0] %}


package:
  name: libgfortran
  # This is hacked to be above the broken one in defaults (which is really libgfortran 2)
  version: 3.0.1
```

This snippet is from the 3.0.1 version on the defaults channels.

Reply:
Ah, makes sense.